app/src/test/java/com/pokerio/app/ServerRequestTest.kt

```diff
@@ -188,20 +226,143 @@ class ServerRequestTest {
 
         server.enqueue(MockResponse().setResponseCode(200))
 
+        var onSuccessCalled = 0
+        val onSuccess = {
+            onSuccessCalled += 1
+        }
+
+        val onError = {
+            assertTrue("JoinGame should not error", false)
```

Comment:
Wrong assert string, it is a test for leaveGame.

Reply:
Good catch!